### PR TITLE
LGPL-3.0

### PR DESCRIPTION
**Pull request by bombcar**
to be tagged after merge

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,67 @@
+modName = FTBLib
+
+# This is a case-sensitive string to identify your mod. Convention is to use lower case.
+modId = FTBL
+
+modGroup = latmod.lib
```
Comment (glowredman) [inline gradle.properties]: Shouldn't this be ftb.lib? https://github.com/GTNewHorizons/FTB-Library/tree/master/src/main/java/ftb/lib

Comment (mitchej123) [reply gradle.properties]: Yah, the latmod just got pulled in instead of having yet another dep.  The main group for this is ftb.lib.

Comment (bombcar) [reply gradle.properties]: was trying to stay as close to https://github.com/GTNewHorizons/FTB-Library/blob/e3019ae0cd62cf83f676284633e4da934fb59d18/build.gradle#L29 as possible without moving files around, perhaps ftb.lib is more correct? It has two folders.

Comment (glowredman) [reply gradle.properties]: I guess LatvianModder interpreted it as maven group and not as path where the mod is

**Review by glowredman — APPROVED** [review]

**Pull request merged**